bindings/wasm/src/did/wasm_document.rs

```diff
@@ -407,78 +402,25 @@ impl WasmDocument {
   /// Creates a signature for the given `data` with the specified DID Document
   /// Verification Method.
   ///
```

Comment:
It was changed to `Vec<u8>` after #734 was merged. I think GitHub might still be having problems since the file changes still show `privateKey: String` but the branch files show `privateKey: Vec<u8>`: https://github.com/iotaledger/identity.rs/blob/28aec0c70f8a7c81d8f92a76d96814c87b774c29/bindings/wasm/src/did/wasm_document.rs#L374